libs/community/langchain_community/embeddings/nemo.py

```diff
@@ -6,6 +6,7 @@
 
 import aiohttp
 import requests
+import warnings
```

Comment:
make sure you `poetry install --with dev,lint,typing,test` and run `make format lint test`